frameworks/PHP/phalcon/public/index-micro.php

```diff
@@ -9,11 +9,13 @@
 
         return new \Phalcon\Db\Adapter\Pdo\Mysql(array(
             'host'       => 'tfb-database',
-            'dbname'       =>  'hello_world',
-            'charset'    =>  'utf8',
+            'dbname'     =>  'hello_world',
             'username'   => 'benchmarkdbuser',
             'password'   => 'benchmarkdbpass',
-            'persistent' => true
+            'options'    => [
```

Comment:
Perhaps now will work the persistent db connection.

Reply:
Could be, docs are still being updated for v4.